File: 0045-jump-game-ii/0045-jump-game-ii.py
class Solution:
    def jump(self, nums: List[int]) -> int:
        pos, count = len(nums) - 1, 0
        while pos > 0:
            for i in range(pos):
                if i + nums[i] >= pos:
                    pos = i
                    count += 1
                    break
        return count

File: 0045-jump-game-ii/0045-jump-game-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var jump = function(nums) {
    let pos = nums.length - 1;
    let count = 0;
    while (pos > 0) {
        for (let i = 0; i < pos; i++) {
            if (i + nums[i] >= pos) {
                pos = i;
                count++;
                break;
            }
        }
        
    }
    return count;
}
    